### ISSUE TITLE
Can't reproduce the stated accuracy 

### ISSUE THREAD

**Issue by qzhai** [RenqiChen/Genomic-Selection#7]
Hi,

I'm a bit confused about the `filter_dim` setting for Rice3k. The original paper states that the SNP sequence lengths for Rice3k and Wheat3k are 201,740 and 404,388, respectively. So, to confirm, I should adjust `filter_dim` rather than `max_len`, correct? Also, could you please share the config file used for training on Rice3k?

I’ve trained the network five times (running `bash distributed_train_rice3k.sh`), but can't reproduce the accuracy, the accuracy for Rice3k remains at 0.

Thanks.

**Comment by RenqiChen** [comment]
Hello, as shown in `dataset_rice3k_45.py` and `model_ker_wheat.py`, `filter_dim` is not employed. You can adjust the `max_len`. I believe the reason why accuracy is 0 is not caused by these parameters. You should print the output step by step to see where the error takes place.

**Comment by qzhai** [comment]
Could you please release the pretrained models? I trained the model using the default script provided, but the accuracy remains at 0.

**Comment by RenqiChen** [comment]
Hello, as written in the paper, it is an end-to-end method. Maybe you can try closing the normalize function and running it again. In our experiments for rice3k, we did not use normalization.

**Comment by RenqiChen** [comment]
I would like to provide the advice that our method is simple, where the key points are random mask and k-mer. Thus you can just use these two techniques in your model to see whether they are effective.

**Comment by RenqiChen** [comment]
Hello, we have revised some errors in the dataset_rice3k_45.py. We hope this can help you to reproduce our results.
